### PR TITLE
Fix `bbolt keys` and `bbolt get` to prevent them from panicking when no parameter provided

### PR DESCRIPTION
**Pull request by Elbehery**
resolves https://github.com/etcd-io/bbolt/issues/681

cc @ahrtr

### CODE REVIEW

**Comment by Elbehery** [comment]
@ahrtr  i will go through all the cmds now 

just this is to get early review

**Comment by Elbehery** [comment]
only `get` & `keys` cmds having this issue .. fixed both 👍🏽

**Comment by Elbehery** [comment]
@ahrtr  fixed, now the output 

- keys cmd
```
 ./bbolt keys
usage: bolt keys PATH [BUCKET...]

Print a list of keys in the given (sub)bucket.
=======

Additional options include:

        --format
                Output format. One of: auto|ascii-encoded|hex|bytes|redacted (default=auto)

Print a list of keys in the given bucket.

Error: not enough arguments.
```

- get cmd
```
 ./bbolt get          
usage: bolt get PATH [BUCKET..] KEY

Print the value of the given key in the given (sub)bucket.

Additional options include:

        --format
                Output format. One of: auto|ascii-encoded|hex|bytes|redacted (default=auto)
        --parse-format
                Input format (of key). One of: ascii-encoded|hex (default=ascii-encoded)"

Error: not enough arguments.

```

File: cmd/bbolt/main.go [hunk 1]
```diff
@@ -921,6 +924,11 @@ func (cmd *keysCommand) Run(args ...string) error {
 
 	// Require database path and bucket.
 	relevantArgs := fs.Args()
+	if len(relevantArgs) < 2 {
+		err := fmt.Errorf("Error: %w.\n", ErrNotEnoughArgs)
+		fmt.Fprintln(cmd.Stderr, cmd.Usage())
+		return err
```
Comment (ahrtr) [inline cmd/bbolt/main.go:930]:
```suggestion
		return ErrNotEnoughArgs
```

File: cmd/bbolt/main.go [hunk 2]
```diff
@@ -1000,6 +1008,11 @@ func (cmd *getCommand) Run(args ...string) error {
 
 	// Require database path, bucket and key.
 	relevantArgs := fs.Args()
+	if len(relevantArgs) < 3 {
+		err := fmt.Errorf("Error: %w.\n", ErrNotEnoughArgs)
+		fmt.Fprintln(cmd.Stderr, cmd.Usage())
+		return err
```
Comment (ahrtr) [inline cmd/bbolt/main.go:1014]:
```suggestion
		return ErrNotEnoughArgs
```

**Comment by ahrtr** [comment]
There are some inconsistency between the legacy commands and cobra style commands, we will need to do some refactor in future after we cut release-1.4 branch.

**Comment by Elbehery** [comment]
sure, i will do the cobra migration as we discussed after the release 👍🏽

File: cmd/bbolt/main.go (hunk 1 above)
Comment (Elbehery) [reply cmd/bbolt/main.go]: updated, but this way the `usage` will not be printed

File: cmd/bbolt/main.go (hunk 2 above)
Comment (Elbehery) [reply cmd/bbolt/main.go]: ditto

File: cmd/bbolt/main.go (hunk 1 above)
Comment (ahrtr) [reply cmd/bbolt/main.go]: Your previous implement (see below) isn't consistent with the cobra style command's output. 

Overall it isn't a big deal for now. Let's refactor it in future.

```
$ ./bbolt keys
usage: bolt keys PATH [BUCKET...]

Print a list of keys in the given (sub)bucket.
=======

Additional options include:

	--format
		Output format. One of: auto|ascii-encoded|hex|bytes|redacted (default=auto)

Print a list of keys in the given bucket.

Error: not enough arguments.
```

**Pull request merged**